hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/crypto/key/KeyProvider.java

```diff
@@ -35,7 +34,6 @@
 import java.util.Map;
 import java.util.Objects;
 
-import org.bouncycastle.jce.provider.BouncyCastleProvider;
 import com.google.gson.stream.JsonReader;
```

Comment:
do we really have to use gson here? i know this is not new, but given we have jackson (and all its update pain) everywhere, it'd be good to be consistent

Reply:
Not sure about gson.  If there is a need, let's fix it separately.